src/Libraries/PythonNodeModelsWpf/PythonNode.cs

```diff
@@ -32,6 +34,29 @@ public void CustomizeView(PythonNode nodeModel, NodeView nodeView)
             var editWindowItem = new MenuItem { Header = PythonNodeModels.Properties.Resources.EditHeader, IsCheckable = false };
             nodeView.MainContextMenu.Items.Add(editWindowItem);
             editWindowItem.Click += delegate { EditScriptContent(); };
+            // If it is a Debug build, display a python engine switcher
+            if (dynamoViewModel.IsDebugBuild)
+            {
+                var pythonEngineVersionMenu = new MenuItem { Header = PythonNodeModels.Properties.Resources.PythonNodeContextMenuEngineSwitcher, IsCheckable = false };
```

Comment:
while this is usually a good thing to do, not sure we need localization to spend time on a debug flag?

Reply:
@mjkkirschner ha, I guess we have been doing it for 3rd party devs from a different country?